install_mac.py

```diff
@@ -47,22 +60,29 @@ def git_clone(repo_url: str, branch='master'):
 def configure_preferences():
     logger.info('configuring preferences')
 
-    # enable execution from everywhere without the annoying "open anyway" from preferences
-    sudo('spctl', '--master-disable')
+    confirm_install('enable execution from everywhere without "open anyway" prompt',
```

Comment:
- this does more then just that
- refactor the messgae: `disable master system policy? (this would ...)`

Reply:
done, open new pr